lib/formatDate.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const twoPaddedNumbers = new Array(60).fill(0).map((v, i) => i.toString().padStart(2, '0'))
+const threePaddedNumbers = new Array(1000).fill(0).map((v, i) => i.toString().padStart(3, '0'))
+
+const monthLookup = new Array(12).fill(0).map((v, i) => (i + 1).toString().padStart(2, '0'))
+const tzOffsetLookup = {}
```

Comment:
Are these constants? Maybe use all capital to indicate they are constants. e.g. `TWO_PADDED_NUMBERS`

Reply:
Done